types/tough-cookie-filestore/index.d.ts

```diff
@@ -0,0 +1,42 @@
+// Type definitions for tough-cookie-filestore 0.0
+// Project: https://github.com/mitsuru/tough-cookie-filestore
+// Definitions by: Christian Friedow <https://github.com/friedow>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface Cookie {
+    domain: string;
+    path: string;
+    key: string;
+}
+
+export default class FileCookieStore {
```

Comment:
It seems like the [underlying library](https://unpkg.com/tough-cookie-filestore@0.0.1/index.js) is not using `export default`, so this .d.ts file should not either. Instead, you should follow the appropriate template from [the TypeScript handbook](http://www.typescriptlang.org/docs/handbook/declaration-files/templates.html). Note that if you want to use `import name from 'package';` in your consuming code, you can use the `esModuleInterop` compiler setting.

Reply:
What exactly would you change here? I mean it still remains a class that gets exported, right?

Reply:
As discussed at that handbook link, I'd do:
```ts
export class FileCookieStore {
// define your class
}

export = FileCookieStore
```
What you have now is a class that gets exported on the `.default` property of the resulting module, which is fine when users of this package have `esModuleInterop` enabled, but if they don't, then it'll cause a runtime error.